search_params.py

```diff
@@ -94,7 +95,13 @@ def init_search_params_spaces(model_config):
                      'qloguniform', 'randn', 'qrandn', 'randint', 'qrandint']
     for key, value in model_config.items():
         if isinstance(value, list) and len(value) >= 2 and value[0] in search_spaces:
-            model_config[key] = getattr(tune, value[0])(*value[1:])
+            search_space, search_args = value[0], value[1:]
+            # If the values in search values are lists (e.g., [2] in [[2], [4], [6]]), the search space must be `grid_search`.
```

Comment:
this comment isn't clear. People ask why we want to use [[2], [4], [6]] instead of [2,4,6]
Need to give an example like what is shown in the previous comment